crates/bevy_winit/src/lib.rs

```diff
@@ -166,9 +167,25 @@ fn change_window(
                         window.set_max_inner_size(Some(max_inner_size));
                     }
                 }
+                bevy_window::WindowCommand::Close => {
+                    let window = winit_windows.remove_window(id);
+                    // Close the window
+                    drop(window);
```

Comment:
Why do we need this explicit `drop` call? Shouldn't this just happen automatically at the end of the scope?

Reply:
I'm not sure. I think I was trying to clarify that the drop is what actually closes the operating system window. Since `Window` owns that window, I thought it better to be explicit about the side effects, and we can confirm that this order is correct.

Reply:
Hmm. I think I'd prefer to just call `winit_windows.remove_window(id)` and have a comment explaining this mechanism.